account/src/main/java/com/user/account/services/UserTransactionService.java

```diff
@@ -21,7 +21,7 @@ public Transactions addTransaction(Transactions newTransaction) {
 	}
 
 	@Override
-	public ArrayList<Transactions> getSummary(long accountNumber) {
+	public ArrayList<Transactions> getSummary(Long accountNumber) {
 		return transactionRepository.findByAccountNumber(accountNumber);
 	}
 
```

Comment:
Put the implementation classes in impl folder for better readability

Reply:
Resolved as recommended